packages/react-components/src/components/SendBox.tsx

```diff
@@ -33,6 +32,10 @@ export interface SendBoxStrings {
    * Placeholder text in SendBox when there is no user input
    */
   placeholderText: string;
+  /**
+   * The warning message when send box text length is more than max limit
+   */
+  textExceedsLimit: string;
```

Comment:
nit: `textTooLong` might be a slightly better name here? Could it be confusing what limit you're talking about? Or is character length the only possible limit here?